rpc/api.proto

```diff
@@ -901,3 +902,11 @@ message CommittedTicketsResponse {
 	}
 	repeated TicketAddress ticketAddresses = 1;
 }
+
+message BestBlockRequest {
+}
+
+message BestBlockResponse {
+	int64 height = 1;
```

Comment:
uint32 (normally i would ask for int32 in Go code, but unsigned is preferred for performance reasons in protobuf).